_posts/2018-04-01-inception-pipelines-pt2.markdown

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title:  "Seeds of Inception - Part 2"
+date:   2018-04-01
+categories: aws continuous deployment cdn SPA cloudfront 
+author: Pete Yandell
+image: img/inception-pipelines/seed_germination.png
+---
+
+**Part 2 sprouting some website goodness** - A practical example for DevOps on AWS
+
+![sprouting some website goodness]({{ site.url }}/img/inception-pipelines/seed_germination_2.png)
+
+## What's the Problem
+
+If you've read my [first post](https://mechanicalrock.github.io//aws/continuous/deployment/2018/03/01/inception-pipelines-pt1) (and you have, haven't you), you either thought "that's absolute crap, why would I bother" or "hey, that's pretty neat, but what can I do with it". If you were the former, then avert your eyes because this post is targeted firmly at the latter.
```

Comment:
Each of the "thoughts" you have needs a `?`

Reply:
Tim made the same comment on the last post. I originally did that, but the markdown linter kept saying to remove them, so I did.

Personally I think they should be in, but maybe leave them out for consistency sake?